pkg/providers/imagefamily/bootstrap/aksbootstrap.go

```diff
@@ -257,15 +257,18 @@ func CredentialProviderURL(kubernetesVersion, arch string) string {
 	// credential provider has its own release outside of k8s version, and there'll be one credential provider binary for each k8s release,
 	// as credential provider release goes with cloud-provider-azure, not every credential provider release will be picked up unless
 	// there are CVE or bug fixes.
-	credentialProviderVersion := "1.29.2"
+	var credentialProviderVersion string
 	switch minorVersion {
 	case 29:
-		credentialProviderVersion = "1.29.2"
+		credentialProviderVersion = "1.29.13"
 	case 30:
-		credentialProviderVersion = "1.30.0"
-
+		credentialProviderVersion = "1.30.10"
 	case 31:
-		credentialProviderVersion = "1.31.0"
+		credentialProviderVersion = "1.31.4"
+	case 32:
+		fallthrough // to default, which is same as latest
```

Comment:
never seen this keyword, this just allows you to declare case <x> -> default?

Reply:
```
In a case or default clause, the last non-empty statement may be a (possibly labeled) "fallthrough" statement to indicate that control should flow from the end of this clause to the first statement of the next clause. Otherwise control flows to the end of the "switch" statement. A "fallthrough" statement may appear as the last statement of all but the last clause of an expression switch.
```